runtime/kusama/src/lib.rs

```diff
@@ -1791,6 +1791,10 @@ sp_api::impl_runtime_apis! {
 			parachains_runtime_api_impl::dmq_contents::<Runtime>(recipient)
 		}
 
+		fn dmq_contents_bounded(recipient: ParaId, start: u32, count: u32) -> Vec<InboundDownwardMessage<BlockNumber>> {
```

Comment:
This probably needs to wait for the next runtime API version and thus blocks on the staging API work.

Reply:
These types of ad-hoc runtime API upgrades are super sketchy